config/utils_methods.py

```diff
@@ -177,4 +177,60 @@ def remove_fields(obj):
             remove_fields(value)
     elif isinstance(obj, list):
         for item in obj:
-            remove_fields(item)
\ No newline at end of file
+            remove_fields(item)
+
+##---------- ONE API - MULTIPLE API CALLS (CRUD OPERATIONS) ---------------
+def add_key_value_to_all_ordereddicts(od_list, key, value):
+    for od in od_list:
```

Comment:
what is od and od_list here, the variable name should be more understandable

Reply:
Updated meaningful variable names